src/collections/_documentation/workflow/integrations/global-integrations.md

```diff
@@ -1136,6 +1136,66 @@ The Split integration is only available for organizations on the Business and En
 Follow the instructions in the link below:  
 <https://help.split.io/hc/en-us/articles/360029879431>
 
+## Deployment
+
+These integrations help automate your release and deployment process.
+
+### Vercel
+
+Vercel is an all-in-one platform with Global CDN supporting static and JAMstack deployment and Serverless Functions. Connect your Sentry and Vercel projects to automatically upload source maps and notify Sentry of release deployment.
+
+**Installation**
+
+1. Visit <https://vercel.com/integrations/sentry/add>
+
+2. Select your account and click "Add".
+
+    [{% asset vercel/vercel_install.png alt="Sentry modal showing Vercel installation." %}]({% asset vercel/vercel_install.png @path %})
+
+    **Note that the installation created an internal integration to generate an auth token used in building releases. If the Vercel internal integration is deleted, your Vercel integration will stop working!**
+
+    [{% asset vercel/vercel_internal_integration.png alt="Sentry modal showing Vercel internal integration." %}]({% asset vercel/vercel_internal_integration.png @path %})
+
+**Project Linking**
+
+1. Select a Sentry project and a Vercel project to link together.
+
+    [{% asset vercel/vercel_link_project.png alt="Sentry modal showing linking Sentry project to Vercel project." %}]({% asset vercel/vercel_link_project.png @path %})
+
+    * This step will generate environment variables in the selected Vercel project. Having these set means you can skip the [CLI Configuration step](https://github.com/getsentry/sentry-webpack-plugin#cli-configuration) in the Sentry Webpack Plugin setup. You can see these in Vercel in **Project Settings** > **General** > **Environment Variables**.
+
+
+    [{% asset vercel/vercel_env_vars.png alt="Sentry modal showing linking Sentry project to Vercel project." %}]({% asset vercel/vercel_env_vars.png @path %})
+
+    * `SENTRY_ORG` - the name of your Sentry organization
+    * `SENTRY_PROJECT` - the name of your linked Sentry project
+    * `SENTRY_AUTH_TOKEN` - the auth token from the Vercel Internal Integration that was created upon installation
+    * `NEXT_PUBLIC_SENTRY_DSN` - the linked Sentry project's DSN
+    * `VERCEL_GITHUB_COMMIT_SHA`, `VERCEL_GITLAB_COMMIT_SHA`, or `VERCEL_BITBUCKET_COMMIT_SHA` - your source code provider's commit SHA, used for creating releases and associating commits
+
+  2. Redeploy your Vercel project in order to trigger a release.
+
+**Usage**
+
+* If you have not already done so, [instrument your code with Sentry]({%- link _documentation/error-reporting/quickstart.md -%}).
+* Ensure you have [installed a repository integration]({%- link _documentation/workflow/releases/index.md -%}#install-repo-integration) and added the relevant repository.
+* Add the [Sentry Webpack Plugin](https://github.com/getsentry/sentry-webpack-plugin) to your configuration. For NextJS, use `next.config.js`.
+
```

Comment:
@ceorourke Why is there an extra line here?

Reply:
I wanted to make them visually separate because the first 3 bullet points address the case where you're brand new to using sentry and sentry-webpack-plugin but have a Vercel project whereas the final one addresses the case where you already have sentry and sentry-webpack-plugin integrated. I was trying to keep it not long winded, but would it be more clear if I added some bold text indicating which situation you may fall into?